config/projects/omnibus-toolchain.rb

```diff
@@ -50,3 +56,18 @@
   signing_identity "Developer ID Installer: Chef Software, Inc. (EU3VF8YLX2)"
 end
 compress :dmg
+
+msi_upgrade_code = "D607A85C-BDFA-4F08-83ED-2ECB4DCD6BC5"
```

Comment:
This code should be unique for this project (can't reuse Chef or ChefDK's). You can generate a new GUID at:
https://www.guidgenerator.com/